Cargo.toml

```diff
@@ -11,3 +11,4 @@ env_logger = "0.10"
 ftdi = "0.1.3"
 log = "0.4.8"
 thiserror = "1.0.15"
+serialport="=4.6.1"
```

Comment:
This should probably be behind a feature, and FTDI can then also go behind a separate feature.